src/slices/BondSlice.ts

```diff
@@ -68,11 +72,11 @@ export interface IBondDetails {
 }
 export const calcBondDetails = createAsyncThunk(
   "bonding/calcBondDetails",
-  async ({ bond, value, provider, networkID }: ICalcBondDetails, { dispatch }): Promise<IBondDetails> => {
+  async ({ bond, value, provider, networkID }: ICalcBondDetails, { dispatch, getState }): Promise<IBondDetails> => {
     if (!value) {
       value = "0";
     }
-
+    console.log("running BondDetails for", bond);
```

Comment:
We need to remove this long line.

Reply:
I removed that in my ENV pr: https://github.com/OlympusDAO/olympus-frontend/pull/490

Reply:
fixed here: https://github.com/OlympusDAO/olympus-frontend/pull/509